package.json

```diff
@@ -129,13 +129,39 @@
 							"description": "Store previously selected Quarkus extensions"
 						}
 					}
-        },
-        "quarkus.tools.symbols.showAsTree": {
-					"type": "boolean",
-					"default": "true",
-					"description": "Show Quarkus properties as tree",
-					"scope": "window"
-				}
+		        },
+		        "quarkus.tools.symbols.showAsTree": {
+						  "type": "boolean",
+						  "default": "true",
+						  "description": "Show Quarkus properties as tree",
+						  "scope": "window"
+		        },
+		        "quarkus.tools.validation.enabled": {
+						  "type": "boolean",
+						  "default": "true",
+						  "description": "Enabled validation",
+						  "scope": "window"
+				},
+		        "quarkus.tools.validation.unknown.severity": {
+						  "type": "string",
+		          "enum": [
+		            "none",
+		            "warning",
+		            "error"
+		          ],
+		          "default": "warning",
+		          "description": "Unknown Quarkus properties severity validation.",
```

Comment:
I think "Validation severity for unknown Quarkus properties" sounds a bit better.

Reply:
@xorye updated